frontend/src/components/simple-auto-filter/simple-auto-filter.tsx

```diff
@@ -30,9 +32,21 @@ const SimpleAutoFilter: FC = () => {
 
   const { filters, brandDetails } = useAppSelector((state) => state.carFilter);
 
+  const {
+    id: detailId,
+    brandId,
+    modelIds,
+  } = useMemo(() => brandDetails[0], [brandDetails[0]]);
```

Comment:
`useMemo` is used to _wrap some calculations or complex evaluation_. And `brandDetails[0], [brandDetails[0]]` is misusing of the `useMemo`